docs/guides/run-capability.md

```diff
@@ -1,33 +1,37 @@
-# Test & run new capability
+# Run Capability
 
-## Setup
+This guide describes how a capability can be used in any production Node.js application, without the use of Superface [remote registry](https://superface.ai/catalog).
```

Comment:
I removed talk about offline use in here since its not really described in guide and added tip about offline use in `Run the app` section. We can add link to offline use guide later.